setup.sh

```diff
@@ -16,6 +16,20 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# Check the repo URL and main branch
+REPO_URL=$(git remote get-url origin)
+if [[ $REPO_URL == *data-team-uhn/cards-project-template.git ]]
```

Comment:
This check should be fixed as, if the repository was cloned with `git clone https://github.com/data-team-uhn/cards-project-template`, `git remote get-url origin` will return `https://github.com/data-team-uhn/cards-project-template` (without the trailing `.git`) and thus the `setup.sh` script will proceed even though the _git remote_ is set to the _template_ repository as opposed to the new _project_ repository.

I think checking if `$REPO_URL` ends with `/cards-project-template` or `/cards-project-template.git` would be sufficient. Including the `data-team-uhn` part of the URL would cause this check to break (always pass) on forks of this repository.

Reply:
Done.